
From: "Antonino A. Daplas" <adaplas@hotpop.com>

Oops, forgot the X86_64 dependency.

1. Make i810fb depend on X86 but not X86_64
2. Fixed typo in i810_init_monspecs(). Reported by Manuel Lauss
   <slauss@resi.at>.

Signed-off-by: Antonino Daplas <adaplas@pol.net>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/video/Kconfig          |    2 +-
 25-akpm/drivers/video/i810/i810_main.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff -puN drivers/video/i810/i810_main.c~i810fb-fixes-2 drivers/video/i810/i810_main.c
--- 25/drivers/video/i810/i810_main.c~i810fb-fixes-2	2004-07-31 17:18:38.050059104 -0700
+++ 25-akpm/drivers/video/i810/i810_main.c	2004-07-31 17:18:38.056058192 -0700
@@ -1656,7 +1656,7 @@ static void __devinit i810_init_monspecs
 		info->monspecs.hfmax = hsync2;
 	if (!info->monspecs.hfmin)
 		info->monspecs.hfmin = hsync1;
-	if (hsync1 < hsync2)
+	if (hsync2 < hsync1)
 		info->monspecs.hfmin = hsync2;
 
 	if (!vsync1)
diff -puN drivers/video/Kconfig~i810fb-fixes-2 drivers/video/Kconfig
--- 25/drivers/video/Kconfig~i810fb-fixes-2	2004-07-31 17:18:38.051058952 -0700
+++ 25-akpm/drivers/video/Kconfig	2004-07-31 17:18:38.057058040 -0700
@@ -457,7 +457,7 @@ config FB_RIVA_DEBUG
 
 config FB_I810
 	tristate "Intel 810/815 support (EXPERIMENTAL)"
-	depends on FB && EXPERIMENTAL && PCI
+	depends on FB && EXPERIMENTAL && PCI && X86 && !X86_64
 	select AGP
 	select AGP_INTEL
 	help
_
